dbms/src/main/java/nl/nn/adapterframework/dbms/DbmsException.java

```diff
@@ -0,0 +1,34 @@
+/*
+   Copyright 2023 WeAreFrank!
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package nl.nn.adapterframework.dbms;
+
+public class DbmsException extends JdbcException {
+	public DbmsException() {
+		super();
+	}
+
+	public DbmsException(String arg1) {
+		super(arg1);
+	}
+
+	public DbmsException(String arg1, Throwable arg2) {
+		super(arg1, arg2);
+	}
+
+	public DbmsException(Throwable arg1) {
+		super(arg1);
+	}
```

Comment:
Nou gebruik ik zelf Eclipse, maar volgens mij probeert IntelliJ altijd de naam van het argument te gebruikt daar waar deze methode aangeroepen wordt, dus daar staat dan new DbmsException( `arg1:` "message hier");
Misschien dat het dan mooier is hier message te gebruiken en in de volgende methodes exception (of e)

Reply:
Oh dit is inderdaad een intelliJ dingetje, ik pas het gelijk aan